python/raft-ann-bench/src/raft-ann-bench/run/conf/wiki_all_1M.json

```diff
@@ -169,7 +170,23 @@
     {
       "name": "raft_cagra.dim32.multi_cta",
       "algo": "raft_cagra",
-      "build_param": { "graph_degree": 32, "intermediate_graph_degree": 48 },
+      "build_param": { "graph_degree": 32,
+        "intermediate_graph_degree": 48,
+        "graph_build_algo": "NN_DESCENT",
+        "ivf_pq_build_pq_dim": 32,
+        "ivf_pq_build_pq_bits": 8,
+        "ivf_pq_build_nlist": 16384,
+        "ivf_pq_build_niter": 10,
+        "ivf_pq_build_ratio": 10,
+        "ivf_pq_search_nprobe": 30,
+        "ivf_pq_search_internalDistanceDtype": "half",
+        "ivf_pq_search_smemLutDtype": "half",
+        "ivf_pq_search_refine_ratio": 8,
+        "nn_descent_max_iterations": 10,
+        "nn_descent_graph_degree": 48,
+        "nn_descent_intermediate_graph_degree": 72,
+        "nn_descent_termination_threshold": 0.001
```

Comment:
These are just examples showing how to specify build params.
- IVF-PQ params are not far from optimal but would need a little more fine tuning. 
- I did not experiment with NN descent parameters.
- Note than `nn_descent_graph_degree` is actually redundant, it has to agree with the `intermediate_graph_deree` (in line 174).

Reply:
I believe @divyegala had the nn descent graph degree being set to the cagra intermediate graph degree in the c++ layer already. If not, we probably want to do this to remove user confusion here.

Reply:
Yes `nn_descent_graph_degree` is correctly set on the C++ side. If user tries to rewrite it, then we emit a warning and restore the correct value. I have removed `nn_descent_graph_degree` from the json config. The cagra parameter tuning guide omits this param.

I would still keep the ability of the json config reader to process this parameter, in case we wan to build simple knn-graphs in the future using nn descent.

Reply:
Yep, that approach sounds great